### PR TITLE
[IMP] theme_*: adapt `s_company_team_detail` to themes

### PR DESCRIPTION
**Pull request by mano-odoo**
*: theme_beauty, theme_bewise, theme_enark, theme_notes, theme_treehouse, theme_yes

This commit adapts the design themes to the new `s_company_team_detail` snippet.

task-4156290
Part of: task-4077427

Community PR: https://github.com/odoo/odoo/pull/179007

### CODE REVIEW

**Review by anso-odoo — CHANGES_REQUESTED** [review]
Hey @mano-odoo small comments for you

File: theme_bewise/views/customizations.xml
[the remark this answers is not in the document]
Comment (mano-odoo) [reply theme_bewise/views/customizations.xml:1]: It was missing the `o_colored_level` class 👍 
I changed the socials to text-o-color-3 for o_cc4 background 
image

**Review by anso-odoo — APPROVED** [review]
LGTM

**Review by bvr-odoo — APPROVED** [review]

**Comment by SergeBayet** [comment]
Commit(s) moved into
- https://github.com/odoo/design-themes/pull/919

**Pull request closed without merging**